src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/TrainingToBeProvidedValidator.cs

```diff
@@ -7,7 +7,7 @@ public partial class CreateApprenticeshipRequestValidator
     {
         private const int TrainingToBeProvidedMaximumLength = 4000;
 
-        private void ConfigureTrainingToBeProvided()
+        private void ConfigureTrainingToBeProvidedValidator()
```

Comment:
❓

Reply:
Renamed method to be consistent with the others